bundler/lib/dependabot/bundler/update_checker/shared_bundler_helpers.rb

```diff
@@ -237,10 +237,6 @@ def using_bundler2?
 
           lockfile.content.match?(/BUNDLED WITH\s+2/m)
         end
-
-        def bundler_version
-          @bundler_version ||= Helpers.bundler_version(lockfile)
-        end
```

Comment:
I've tentatively removed this as I think all the classes mixing it in are overriding it locally, but CI may prove me wrong